### DISCUSSION TITLE
Playlist view with separation by Album (for example)

### DISCUSSION THREAD

**Discussion by lordoff** [strawberrymusicplayer/strawberry#834] [General]
Hello,
I liked very mush Strawberry. Give me everything, that is needed from one good developed player.
But almost...
Its possible in any way in Playlist to make visual separation by Albums (for example). 
Similar like Cantata or even better, deadbeef. I tried to find such option, but no success.
I attach screenshots for comparison.

![Screenshot_deadbeef](https://user-images.githubusercontent.com/23076154/142739174-1c4192b9-2152-4211-af62-4d040255c9a3.png)

![Screenshot_Cantata](https://user-images.githubusercontent.com/23076154/142739179-76de77ba-25a4-473d-91a5-6d1a2d6bb7a5.png)

**Comment by jonaski** [comment]
Add this to https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
